client/src/components/landing/LandingNavigationBar.vue

```diff
@@ -51,9 +53,13 @@ export default Vue.extend({
   &-buttons {
     display: flex;
 
-    > :not(:last-child) {
+    > * {
```

Comment:
Why last child should have margin right?

Reply:
Szymons request to be margin in the end